app/helpers/admin/editions_helper.rb

```diff
@@ -279,8 +283,8 @@ def translation_preview_links(edition)
     end
   end
 
-  def archiving_or_unpublishing(edition)
-    edition.unpublishing.unpublishing_reason_id == UnpublishingReason::Archived.id ? 'archiving' : 'unpublishing'
+  def withdrawal_or_unpublishing(edition)
+    edition.unpublishing.unpublishing_reason_id == UnpublishingReason::Withdrawn.id ? 'withdrawal' : 'unpublishing'
```

Comment:
Yeah, my thinking was that we have a better option with "withdraw" than adding an "ing" onto the end, i.e. the thing that represents the outcome of withdrawing a document is a withdrawal. There isn't an equivalent noun for unpublishing or archiving.